source/slang/hlsl.meta.slang

```diff
@@ -121,13 +121,13 @@ struct ByteAddressBuffer
     [__readNone]
     [ForceInline]
     [require(cpp_cuda_glsl_hlsl_metal_spirv, byteaddressbuffer)]
-    uint2 Load2(int location)
+    uint2 Load2(int location, int aligned)
```

Comment:
For Load2/Load3/Load4/Load<T>, we want to also add:
Load2Aligned/Load3Aligned/Load4Aligned/LoadAligned<T> in parallel to this methods and pass `1` to `aligned` parameter when calling `__byteAddressBufferLoad`.

Reply:
done